dist/tools/flake8/check.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
why bash?

Reply:
because of the colored output when errors are found (using warning color). I couldn't make it work with `sh` without having `tput` result in the terminal... Maybe you have an idea ?

Reply:
ok